pkg/controller/workload/workload_controller.go

```diff
@@ -47,11 +49,13 @@ func NewController(bpfWorkload *bpf.BpfKmeshWorkload) *Controller {
 		bpfWorkloadObj: bpfWorkload,
 	}
 	c.Rbac = auth.NewRbac(c.Processor.WorkloadCache)
+	c.Telemetry = telemetry.NewMetric(c.Processor.WorkloadCache, bpfWorkload)
 	return c
 }
 
 func (c *Controller) Run(ctx context.Context) {
 	go c.Rbac.Run(ctx, c.bpfWorkloadObj.SockOps.MapOfTuple, c.bpfWorkloadObj.XdpAuth.MapOfAuth)
+	go c.Telemetry.Run(ctx, c.bpfWorkloadObj.SockConn.MapOfMetricNotify, c.bpfWorkloadObj.SockConn.MapOfMetrics)
```

Comment:
nit: since c.Telemetry contains bpfWorkload, we donot need to pass the maps here